packages/runner/src/crawler/hc-crawler-options-factory.ts

```diff
@@ -48,7 +46,7 @@ export class HCCrawlerOptionsFactory {
             retryCount: 1,
             preRequest: (options: CrawlerRequestOptions) => {
                 let processUrl = true;
-                if (options.url.indexOf('https://login.microsoftonline.com/') !== -1 || options.url.match(IGNORED_EXTENSIONS) !== null) {
+                if (!this.isAllowedUrl(options.url)) {
```

Comment:
> (!this.isAllowedUrl(options.url)) { [](start = 18, length = 36)

this should also check for node_url.parse(link).hostname
(currently this is not an issue, since depth level is 1)